src/test/HelmetTest.js

```diff
@@ -217,7 +217,20 @@ describe("Helmet", () => {
 
                 const htmlTag = document.getElementsByTagName("html")[0];
 
-                expect(htmlTag.attributes.length).to.equal(0);
+                expect(htmlTag.getAttribute("lang")).to.be.null;
+                expect(htmlTag.getAttribute("amp")).to.be.null;
```

Comment:
Can you also test that Helmet overrides attributes handled outside of Helmet when argument is present in Helmet

```
const htmlTag = document.getElementsByTagName("html")[0];
htmlTag.setAttribute("lang", "en-us");
<Helmet
    htmlAttributes={{
        "lang": "en-gb"
    }}
/>

expect(htmlTag.getAttribute("lang")).to.be.equal("en-gb");
```

And test that Helmet does not override attributes when argument not present in Helmet

```
const htmlTag = document.getElementsByTagName("html")[0];
htmlTag.setAttribute("lang", "en-us");
<Helmet />
expect(htmlTag.getAttribute("lang")).to.be.equal("en-us");
```